pipeline_dp/pipeline_backend.py

```diff
@@ -377,6 +393,10 @@ def map(self, rdd, fn, stage_name: str = None):
             return self._sc.parallelize(rdd).map(fn)
         return rdd.map(fn)
 
+    def map_with_side_inputs(self, rdd, fn, side_input_cols, stage_name: str):
+        raise NotImplementedError("map_with_side_inputs "
```

Comment:
I tried to find but didn't find any information how side inputs work in apache spark and whether they are supported at all. Do you know how it works in Spark?